Sealer/sealer.cc

```diff
@@ -603,38 +602,48 @@ void kRun(const ConnectPairsParams& params,
 
 	printLog(logStream, "Flanks inserted into k run = " + IntToString(flanks.size()) + "\n");
 
-	for (read1_it = flanks.begin(); read1_it != flanks.end();) {
-		success = false;
+	int counter = 0;
+	vector<map<FastaRecord, map<FastaRecord, Gap> >::iterator> flanks_closed;
+#pragma omp parallel private(read1_it, read2_it) firstprivate(counter)
+	for (read1_it = flanks.begin(); read1_it != flanks.end(); ++read1_it) {
 		FastaRecord read1 = read1_it->first;
-		for (read2_it = flanks[read1].begin(); read2_it != flanks[read1].end(); read2_it++) {
+		bool success = false;
+		for (read2_it = flanks[read1].begin(); read2_it != flanks[read1].end(); ++read2_it, ++counter) {
+#if _OPENMP
+			if (counter % omp_get_num_threads() != omp_get_thread_num())
```

Comment:
Do you know whether these function calls are faster? Should their result be explicitly cached? I don't know the answer. I prefer this code as is if they are fast (ideally inline).

Reply:
Acording to https://stackoverflow.com/a/22962390 
It should be slightly faster to explicitly cache it, because the functions aren't inline, so I'll see to add that.

Reply:
I'm okay with the way it is now.